app/app.go

```diff
@@ -1021,19 +1021,4 @@ func (app *PstakeApp) RegisterUpgradeHandler() {
 			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 		},
 	)
-
-	upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
-	if err != nil {
-		panic(err)
-	}
-
-	if upgradeInfo.Name == UpgradeName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
-		storeUpgrades := store.StoreUpgrades{
-			Added:   []string{ratesynctypes.StoreKey, liquidstaketypes.StoreKey},
```

Comment:
keep this blob in general, just remove the `Added: ` things.. 
anyway everything is under if statements, it won't trigger unless there is an upgrade.